application/classes/Ushahidi/Core.php

```diff
@@ -395,6 +397,7 @@ public static function init()
 
 		$di->setter['Ushahidi_Formatter_Set']['setAuth'] = $di->lazyGet("authorizer.set");
 		$di->setter['Ushahidi_Formatter_CSV']['setAuth'] = $di->lazyGet("authorizer.csv");
+        $di->setter['Ushahidi_Formatter_CountryCode']['setAuth'] = $di->lazyGet("authorizer.country_code");
```

Comment:
indent

Reply:
🙈

Reply:
Hm, it looks like its only githubs ui acting up, its not indented wrongly in the file itself